### DISCUSSION TITLE
[Feature request]: Possibility to disable the recent messages screens

### DISCUSSION THREAD

**Discussion by macvenez** [meshtastic/firmware#4654] [General]
Hi!
It may be me but it's not a feature I use at all also because I usually use my device (Heltec V3) connected to my phone.
Would it be possible to add an option do disable all screen regarding recent messages (maybe also from the Android and iOS app) or maybe limit those to the last n ones?
It sometimes isn't always easy to navigate through those menus as the button click is not always received and there may be many messages.

Thanks!

**Comment by overkillthemighty** [comment]
Agreed. I would like it if the screen had an option for just device status. Maybe make an indicator that a new message was just received or something, but usually when I am pressing the button, I want to know what the battery status is.

**Reply by macvenez** [reply]
Same for me 95% of the time I use the button to navigate to see GPS status or battery

**Comment by mariotti** [comment]
Hoi, is this device specific?

**Comment by alternativesurfer** [comment]
Yes please!
